src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/repository/TaskResourceRepository.java

```diff
@@ -29,8 +29,8 @@ public interface TaskResourceRepository extends CrudRepository<TaskResource, Str
     String CHECK_PUBLICATION =
         "select count(*) from pg_publication pgp WHERE pubname='task_publication';";
 
-    String CREATE_PUBLICATION = "CREATE PUBLICATION task_publication FOR TABLE cft_task_db.tasks "
-        + "WITH (publish = 'insert,update,delete');";
+    String CREATE_PUBLICATION = "CREATE PUBLICATION task_publication FOR TABLE cft_task_db.tasks, "
+        + "cft_task_db.work_types WITH (publish = 'insert,update,delete');";
```

Comment:
It is good to use {h-schema} to access the default schema name configured in spring.jpa.hibernate.default_schema. This queary would not work if spring configuration overridden